content/en/blog/_posts/2024-secret-sync-controller-alpha.md

```diff
@@ -0,0 +1,56 @@
+---
+
+The [Secret Sync Controller](https://github.com/kubernetes-sigs/secrets-store-csi-driver/tree/feature/secrets-sync-controller) (SSC) provides cluster admins with a unified way of synchronizing cloud-stored secrets as Kubernetes secrets. This enables workloads to read these secrets via standard Kubernetes mechanisms, even with intermittent connectivity. The SSC will sit alongside the [existing secret store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver). The SSC will eventually replace the CSI driver's sync as Kubernetes secret feature. Still, the CSI driver will remain the recommended solution for cloud scenarios where connectivity is not a concern and persisting secrets within the cluster (syncing as Kubernetes secrets) is not desired. The SSC is an alpha feature and can be used with any [supported cloud provider](https://github.com/kubernetes-sigs/secrets-store-csi-driver).
+
+## What is the motivation for this feature?  
+Many production-scale Kubernetes solutions use secrets stored in an external secret store such as a cloud provider secret management system. These solutions face the same problem: securely accessing those secrets within a Kubernetes cluster. Leveraging a secret store CSI driver allows access to secrets while a cluster is online; however, until now, the secret store CSI Driver has yet to provide a unified solution for access to external secrets while offline. The Secret Sync Controller provides a unified way of pulling secrets from a cloud provider and storing them in the Kubernetes secret store for offline/intermittent access. 
+
+## What is the benefit?  
+1. Offline Access: Many clusters running in production environments need to be resilient to intermittent connectivity. This includes continuing to have access to secrets when a cluster goes offline and a pod restarts. Workloads can continue to access secrets by storing secrets in the Kubernetes secret store instead of a directly mounted volume (which is how the existing secret store CSI driver enables accessing secrets). 
+
+2. Standard K8s Secret Access: Secrets can be accessed via volume mounting, environment variables, or via the Kubernetes API. Workloads and ingress controllers do not need to be customized to access a cloud provider secret store and developers have options on how to access secrets.  
+
+3. Security: The Secret Sync Controller (SSC) has limited permissions and leverages the latest Kubernetes security features (complete list below) so that cluster admins do not need to configure and restrict permissions themselves.  
+
+## How to use the Secret Sync Controller 
+The Secret Sync Controller (SSC) must be used alongside a [secret store Cloud Provider](https://secrets-store-csi-driver.sigs.k8s.io/getting-started/installation.html) that implements the Secret Store CSI Driver provider interface to provide an end-to-end solution for synchronizing secrets from the cloud secret store to the Kubernetes secret store.  
```

Comment:
Use https://secrets-store-csi-driver.sigs.k8s.io/getting-started/installation.html#install-external-secret-providers instead of https://secrets-store-csi-driver.sigs.k8s.io/getting-started/installation.html